app/views/spree/taxons/_filters.html.erb

```diff
@@ -24,11 +24,13 @@
         </div>
       </div>
     </div>
-    <div class="col-xl-5 col-lg-7" data-hook="taxon_products">
```

Comment:
why we removed this data hook : data-hook="taxon_products". Is there any specific reason. Please justify